pygeoogc/__init__.py,sha256=MKCswY83wMLc7EMVsE5UTU60PCdBHP8uRcT825tJMtc,318
pygeoogc/exceptions.py,sha256=moWPK_rK2jWWB3HI7eB_U9oPhKoG107--4osAZnAZeg,1969
pygeoogc/pygeoogc.py,sha256=AKctU_aeTc-5bj2v0AeKgtkGVxpgbpgMZpB3waIC99w,26613
pygeoogc/utils.py,sha256=3_z2sYwM6vLTMZlJW0220V3GLsbwZk42YfxHvAIwLCo,10105
pygeoogc-0.1.2.dist-info/AUTHORS.rst,sha256=FAneYiHTTZuEAtX7AvBFFp5_QesoMSH0cUAt3tirVYQ,157
pygeoogc-0.1.2.dist-info/LICENSE,sha256=EM87NgYgabO0a_E-z2nmbsrxomXdI5BDCgmomVDvLsE,1096
pygeoogc-0.1.2.dist-info/METADATA,sha256=UpZnzL8jf6wx8i67kGDP-j4jTCf1ZbMwn7IwjBaCPBo,5222
pygeoogc-0.1.2.dist-info/WHEEL,sha256=kGT74LWyRUZrL4VgLh6_g12IeVl_9u9ZVhadrgXZUEY,110
pygeoogc-0.1.2.dist-info/top_level.txt,sha256=LlCRBzIb5C1PpXavuTkqP7BhlJHJARnYgOzJFpXmxms,9
pygeoogc-0.1.2.dist-info/RECORD,,
